includes/admin/transifex-live-integration-admin.php

```diff
@@ -315,4 +318,33 @@ static public function sanitize_settings( $settings ) {
 		return $settings;
 	}
 
-}
+	/**
+	 * Function that makes a (dummy) ping call to the Transifex API, in order to keep
+	 * information about the Wordpress instance. This information will help us
+	 * identify and provide better support for specific versions.
+	 *
+	 * The actual API endpoint doesn't matter (in fact the provided api key is Live
+	 * specific and can't be used for an API call), we are only interested in the
+	 * request itself.
+	 *
+	 * @param string $api_key
+	 */
+	static public function ping_tx($api_key){
+	    global $wp_version;
+
+		$agent = array(
+		 "transifex-live-wordpress-version/" . $wp_version,
+		 "transifex-live-wordpress-user/" . md5($api_key)
```

Comment:
If the API key doesn't work, do we really need it in the request?

Reply:
@dbendilas what do you mean by doesn't work? Being invalid? The integration always assumes that there is a valid API key in place (otherwise the save button is not enabled -- the actual check for the valid API key takes place in JS).